ironfish/src/primitives/spend.ts

```diff
@@ -4,7 +4,7 @@
 
 import { Nullifier } from './nullifier'
 
-export const SPEND_SERIALIZED_SIZE_IN_BYTE = 388
+export const SPEND_SERIALIZED_SIZE_IN_BYTE = 356
```

Comment:
This number was wrong.

Reply:
Yeah, that's not ideal. I filed a ticket to look at these later https://linear.app/if-labs/issue/IFL-764/const-cleanup

Reply:
Can you list out the calculation in the comment like what we have in the existing code so that we can keep this value up to date?

Reply:
// 32 nullifier
// 4 size
// 160?? commitment